benchmarks/disagg_benchmarks/disagg_prefill_proxy_server.py

```diff
@@ -1,63 +1,196 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: Copyright contributors to the vLLM project
 
+import asyncio
+import logging
 import os
+import time
+from collections import deque
 
 import aiohttp
-from quart import Quart, make_response, request
+from quart import Quart, Response, make_response, request
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# Timeout for backend service requests (seconds)
+AIOHTTP_TIMEOUT = aiohttp.ClientTimeout(total=300)
+# Maximum concurrent requests to backend services
+MAX_CONCURRENT_REQUESTS = 100
+REQUEST_QUEUE_SIZE = 500  # Maximum number of requests in the queue
+RATE_LIMIT = 40  # Maximum requests per second (rate limiting)
+# Prefill service endpoint
+PRE_SERVICE_URL = "http://localhost:8100/v1/completions"
```

Comment:
```suggestion
PREFILL_SERVICE_URL = "http://localhost:8100/v1/completions"
```

I would rename this to be consistent with the decode one